test/Confluent.Kafka.IntegrationTests/Tests/Producer_ProduceAsync_DeliveryHandler.cs

```diff
@@ -50,9 +51,8 @@ public void HandleDeliveryReport(Message dr)
                 Assert.Equal(Topic, dr.Topic);
                 Assert.True(dr.Offset >= 0);
                 Assert.Equal(TimestampType.CreateTime, dr.Timestamp.Type);
-                Assert.True(Math.Abs((DateTime.UtcNow - dr.Timestamp.DateTime).TotalMinutes) < 1.0);
 
-                if (Count < 5)
+                if (Count != 5)
                 {
                     Assert.Equal(TestKey, dr.Key);
```

Comment:
I don't seem to receive the handler in order. They are fired i different thread (note that i didn't run the bootstrap.sh and had a 3 partition topic) and even locking didn't solve the problem
Doing this test on a 1 partition topic doesn't seem to be right as we test partitioning